ide/provisioning/install-android-tools.sh

```diff
@@ -32,10 +37,12 @@ else
 fi
 echo
 
-cd ~/opt/android-sdk-linux
+cd ${ANDROID_SDK_DIR}
 echo Installing Android SDK packages...
-tools/bin/sdkmanager "build-tools;${ANDROID_BUILD_TOOLS_VERSION}" \
-    "platforms;android-${ANDROID_PLATFORM_VERSION}"
+echo cmdline-tools/bin/sdkmanager --sdk_root=${ANDROID_SDK_DIR} \
+    "build-tools;${ANDROID_BUILD_TOOLS_VERSION}" "platforms;android-${ANDROID_PLATFORM_VERSION}"
```

Comment:
This is rather pointless. Anybody who wishes to see the commands can use `-x`.

Reply:
Sorry for that - I'm coming from windows - and didn't know the option  -x. Of course this seems to be much better (readable).